### PR TITLE
We have new Core Contributors!

### PR DESCRIPTION
**Pull request by otaj**
## What does this PR do?

Adds new core contributors to the docs

cc @Atharva-Phatak @matsumotosan @lijm1358 @Borda

### CODE REVIEW

**Review by ethanwharris — APPROVED** [review]
🎉

File: docs/source/governance.rst [hunk 1]
```diff
@@ -8,10 +8,16 @@ Core Maintainers
...
+-----------------
+- Atharva Phatak (`Atharva-Phatak <https://github.com/Atharva-Phatak>`_)
+- Shion Matsumoto (`matsumotosan <https://github.com/matsumotosan>`_)
+- JongMok Lee (`lijm1358 <https://github.com/lijm1358>`_)
 
 Alumni
 ------
-- Teddy Koker (`teddykoker <https://github.com/teddykoker>`_)
-- Annika Brundyn (`annikabrundyn <https://github.com/annikabrundyn>`_)
+- Akihiro Nitta (`akihironitta <https://github.com/akihironitta>`_)
```
Comment (Borda) [inline docs/source/governance.rst:20]: @akihironitta are you fine with becoming alumni or will you stay around? :)

Comment (akihironitta) [reply docs/source/governance.rst]: Feel free to revert b9ac91e to drop me.

**Review by akihironitta — APPROVED** [review]
🎉

File: docs/source/governance.rst (hunk 1 above)
Comment (otaj) [reply docs/source/governance.rst]: Oh, we'd like to keep you around :zap:

**Pull request merged**